python/lsst/rubintv/production/starTracker.py

```diff
@@ -476,19 +525,27 @@ def runAnalysis(self, exp, filename):
         oldAz = geom.Angle(oldAz, geom.degrees)
         oldAlt = geom.Angle(oldAlt, geom.degrees)
 
-        # TODO: DM-38889: Remove overrides when weather station is publishing
-        # The override values being supplied here are the nominal values that
-        # the pointing component uses when there is no weather data available.
-        # Once the weather station is providing these and the visitInfo is
-        # being populated, these values should be removed, but only once
-        # they're confirmed to also be making it to the pointing component, in
-        # order that things remain consistent.
+        pressure = exp.visitInfo.weather.getAirPressure()
+        if not np.isfinite(pressure):
+            self.log.warning("Pressure not found in header, falling back nominal value=0.770 bar")
+            pressure = 0.770
```

Comment:
Technically, it's unitless (because it's in C++), and what you get depends on how the translator for that particular thing is written, _i.e._ it's possible for it to change behaviour wrt different cameras/obs_packages (AuxTel has an `if >10000` safety check btw). Obviously the intent is static though, but I'm reading these exposures raw, and what I get back comes out in the 0-1 range, so this is correct while we're doing it without a butler. So that doesn't mean that AP needs to change, necessarily (though I would perhaps check).